src/Symfony/Bundle/FrameworkBundle/Tests/Kernel/flex-style/src/FlexStyleMicroKernel.php

```diff
@@ -69,7 +69,7 @@ public function __sleep(): array
         throw new \BadMethodCallException('Cannot serialize '.__CLASS__);
     }
 
-    public function __wakeup()
+    public function __wakeup(): void
```

Comment:
```suggestion
    public function __wakeup(): never
```

Reply:
not needed, wouldn't provide anything, better be borring.